### PR TITLE
ui: Bootstrap 5 leftover design fixes

### PR DESCRIPTION
**Pull request by thi4go**
This diff fixes some leftover design issues caused by the bootstrap 5
package upgrade.

### CODE REVIEW

File: cmd/dcrdata/public/scss/utils.scss [hunk 1]
```diff
@@ -158,6 +162,11 @@
   padding: 0.3rem 0;
 }
 
+.px7-5 {
```
Comment (thi4go) [inline cmd/dcrdata/public/scss/utils.scss:165]: It seems like previous bootstrap (< 5) applied a left/right 7.5 padding to the `col` class. This helper is used to tilt back the `Mempool` homepage section to where it originally were.

File: cmd/dcrdata/public/scss/bootstrap_overrides.scss [hunk 2]
```diff
@@ -16,3 +16,7 @@
     overflow: visible;
   }
 }
+
+.small {
+  font-size: 80%;
+}
```
Comment (thi4go) [inline cmd/dcrdata/public/scss/bootstrap_overrides.scss:22]: The `small` bootstrap class for texts also changed. The new version 5 one applies a `font-size: .875em;`. This overrides it to the previous setting that dcrdata used.

File: cmd/dcrdata/views/ticketpool.tmpl [hunk 3]
```diff
@@ -37,7 +37,7 @@
         <br>
         <div class="justify-content-between">
           <div class="dygraph-label dygraph-title mb-1">Distribution of Tickets by Reward Outputs</div>
-          <div class="row col-lg-12 col-sm-8 d-flex text-center m-auto">
+          <div class="col d-flex text-center mw50 w-100 m-auto ">
```
Comment (thi4go) [inline cmd/dcrdata/views/ticketpool.tmpl:40]: `col-lg-12` used to apply a `max-width: 50%` setting, which changed to `width: 50%` with the update. This sets it back to how dcrdata used it with the `mw50` helper

**Review by chappjc — APPROVED** [review]
Formatting matches perfectly.
TY for using _variables.scss instead of overrides.

**Pull request merged**